v2/tools/generator/internal/codegen/storage/conversion_graph_test.go

```diff
@@ -144,7 +144,7 @@ func Test_ConversionGraph_WhenRenameConfigured_FindsRenamedType(t *testing.T) {
 	graph, err := builder.Build()
```

Comment:
Missing t.Parallel here too?

I wonder if there's a linter or tool we can run?

Reply:
I also realized I missed it in my new tests for one of my active PRs.

Reply:
There is a linter here: https://github.com/kunwardeep/paralleltest - I'll see if I can turn it on